superset/views/core.py

```diff
@@ -1690,6 +1690,7 @@ def dashboard(**_: Any) -> None:
 
     @api
     @event_logger.log_this
+    @has_access
```

Comment:
I'm seeing a regression with this change. I'm getting auth failures. Changing this to @has_access_api fixes the issue